src/components/Table/Table.tsx

```diff
@@ -209,12 +208,20 @@ const TableRow = styled.tr<TableRowProps>`
   }
 `;
 
-const TableData = styled.td<{ $size: TableSize }>`
+interface TableDataProps {
+  $size: TableSize;
+  $isDeleted?: boolean;
+  $isDisabled?: boolean;
```

Comment:
From a component interface perspective, do we need both of these properties? `isDeleted` seems kind of implementation specific. Couldn't `isDisabled` handle that? I'm not seeing `isDeleted` used on tables in CP.

Maybe @vineethasok or @gjones could answer

Reply:
Agreed, I'd try to keep it as simple as possible until there is a clear reason why we need a separate style between `isDeleted` and `isDisabled`, and if we decide there's a legit reason, they should look distinct from one another.